app/kitchensink/index.html

```diff
@@ -2430,233 +2430,59 @@ <h2 id="Tables" class="page-header">Tables
               <div class="panel">
                 <div class="panel-body">
                   <h3>CBP Default Table</h3>
-                  <h4>Includes Borderless and Hover stylings with MDL additions <small>Just by using the classnames: "table mdl-js-data-table"</small></h4>
-                  
-                    <!-- <table class="table mdl-js-data-table">
-                      <caption>Standard Table</caption>
-                      <thead>
-                        <tr>
-                          <th scope="col" class="non-numeric">Project name</th>
-                          <th scope="col" class="non-numeric">Description</th>
-                          <th scope="col">Price</th>
-                        </tr>
-                      </thead>
-                      <tbody>
-                        <tr>
-                          <th scope="row" class="non-numeric">Invoice</th>
-                          <td class="non-numeric">
-                            <span>Add invoice confirmation</span>
-                          </td>
-                          <td>
-                            <span>$4,500</span>
-                          </td>
-                        </tr>
-                        <tr>
-                          <th scope="row" class="non-numeric">Templates</th>
-                          <td class="non-numeric">
-                            <span>Refactor templates to use the new templating library</span>
-                          </td>
-                          <td>
-                            <span>$9,000</span>
-                          </td>
-                        </tr>
-                        <tr>
-                          <th scope="row" class="non-numeric">Payments</th>
-                          <td class="non-numeric">
-                            <span>Process payments using third party api</span>
-                          </td>
-                          <td>
-                            <span>$5,400</span>
-                          </td>
-                        </tr>
-                        <tr>
-                          <th scope="row" class="non-numeric">Account Management</th>
-                          <td class="non-numeric">
-                            <span>Improve account management workflow</span>
-                          </td>
-                          <td>
-                            <span>$6,000</span>
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table> -->
+                  <h4>Includes Borderless and Hover stylings with MDL additions <small>Just by using the classnames: "table mdl-js-data-table"</small></h4>                  
+                   <!--  <table class="table mdl-js-data-table">
```

Comment:
Why are we removing a table example?

Reply:
It was commented out. I removed the comments and checked in the changes.